lib/src/index.js

```diff
@@ -46,7 +49,12 @@ function src(glob, opt) {
       .pipe(getContents(options));
   }
 
-  return outputStream.pipe(pass);
+  if (options.passthrough) {
+    var inputPass = through.obj();
```

Comment:
What linter? The project doesn't have JSCS, and JSHint has moved away from checking code style I believe. Anyway, should I move the `var` declarations to the top of their scope?